modules/security/oidc-proxy/README.md

```diff
@@ -0,0 +1,53 @@
+# OIDC Proxy Module
+
+The OIDC Proxy is a reverse proxy for Kubernetes authentication via and external provider using the OIDC standard,
+that can be used when you cannot setup the api-server directly, like with managed kubernetes services like GKE, AKS etc.
```

Comment:
```suggestion
The OIDC Proxy is a reverse proxy for Kubernetes authentication via an external provider using the OIDC standard.
It can be used when you cannot set up the API server directly, like with managed Kubernetes services such as GKE, AKS, etc.
```